tests/hedgehog/add_layer_to_allure.py

```diff
@@ -0,0 +1,80 @@
+import sys
+import argparse
+import yaml
+import json
+
+from os.path import exists
+from pathlib import Path
+
+
+def read_yaml(path):
```

Comment:
Probably would be great to extract `read_yaml/read_json` methods into separate file, for example `hh_helpers.py`.
It is not related to this PR, I am just thinking out of loud

Reply:
If move `read_yaml/read_json` methods into a separate file,  can't use ./add_layer_to_allure.py as an independent script.
I think better to open a new PR with moving all common logic into a separate file.